packages/familie-form-elements/src/ja-nei-spørsmål/jaNeiSpørsmål.stories.tsx

```diff
@@ -1,26 +1,55 @@
-import React from 'react';
+import React from "react";
 import { JaNeiSpørsmål } from './JaNeiSpørsmål';
 import { ESvar } from './typer';
+import styled from "styled-components";
+import { number, withKnobs } from "@storybook/addon-knobs";
+import { NumberTypeKnobOptions } from "@storybook/addon-knobs/dist/components/types";
 
 export default {
     component: JaNeiSpørsmål,
     parameters: {
         componentSubtitle: 'En toggle for å svare på ja- og neispørsmål',
     },
     title: 'Komponenter/Form-elementer/JaNeiSpørsmål',
+    decorators: [withKnobs]
 };
 
+const DivMedBredde = styled.div<{bredde: string}>`
+  width: ${props => props.bredde};
+`
+
+const JaNeiMedBreddeToggle: React.FC = ({children}) => {
+  const knobOptions: NumberTypeKnobOptions = {
```

Comment:
Hva er dette? :D aldri sett før

Reply:
Ja! Det er ganske fett, lerna har støtte for at man kan tweake på variabler til komponenter. Her har jeg brukt det bare slik at man kan endre bredde på containeren fordi ja/nei i 100% bredde på desktop ser litt rart ut, men man kan basically gjøre slik at man kan konfigurere props i storybook frontend. Skulle legge ved screenshots men nå har det bugga seg hos meg, undersøker hvorfor.

Reply:
![image](https://user-images.githubusercontent.com/77009351/114529766-95393780-9c4a-11eb-8b91-f13cacb2ea91.png)
Sånn ser det ut. Issuet jeg har fikses hvis jeg fjerner knobs som devDependency for familie-form-elements og bare har det på workspace-nivå, men da skriker eslint til meg om no implicit dependencies... :upside_down_face: